### PR TITLE
ISPN 5830  - Backup sites operations

### PR DESCRIPTION
**Pull request by pzapataf**

### CODE REVIEW

File: src/main/webapp/cluster-view/cluster-view.controller.js [hunk 1]
```diff
@@ -186,6 +186,142 @@ var app = angular.module('managementConsole')
...
+          $scope.successExecuteOperation = false;
+          $scope.errorExecuting = false;
+          $scope.errorDescription = null;
+
+          // Confirms and executes a site operation
+          $scope.executeSiteOperation = function(siteName, operationId, confirmationMessage) {
+
+              // Get confirmation dialog
+              var confirmDialog = $modal.open({
+                  templateUrl: '/cluster-view/confirmation-message-modal.html',
```
Comment (tristantarrant) [inline src/main/webapp/cluster-view/cluster-view.controller.js]: Template URLs need to be relative. Remove the first slash.

File: src/main/webapp/cluster-view/cluster-view.controller.js [hunk 2]
```diff
@@ -186,6 +186,142 @@ var app = angular.module('managementConsole')
...
+
+          $scope.refresh();
+      }
+
+      /// End of site management dialog controller
+
+      // Opens the site management dialog
+      $scope.openSiteDialog = function() {
+          return $modal.open({
+              templateUrl: '/cluster-view/manage-sites-modal.html',
```
Comment (tristantarrant) [inline src/main/webapp/cluster-view/cluster-view.controller.js]: Remove the first slash

**Comment by pzapataf** [comment]
Fixed  template paths

**Comment by vblagoje** [comment]
I'll integrate @pzapataf PRs @tristantarrant

**Comment by vblagoje** [comment]
Integrated as one squashed commit - 669afc294ec0c985a91ba74b01c5eccf42a8146b

**Pull request closed without merging**